### ISSUE TITLE
App Store submission error ITMS-90725 (iOS SDK 13 or later)

### ISSUE THREAD

**Issue by FrogDigital** [Gamua/Adobe-Runtime-Support#410]
I feel sorry for posting this as Im sure it has already been fully dealt with elsewhere, but I tried and tried to find a solution on this forum and others but alas no definitive answer.

The problem:
I submitted an update to an app I created in Adobe Animate 2019 (Windows 10), using AIR 33.1.1.63.
The app was refused by Apple, who cited this error message:
_ERROR ITMS-90725: "SDK Version Issue. This app was built with the iOS 12.1 SDK. As of June 30, 2020, all apps for iPhone or iPad must be built with the iOS 13 SDK or later, included with Xcode 11 or later"._

I am unsure how I can use Adobe Animate to create a new version of my app which will use iOS 13 SDK.

If I update to the latest Harman AIR 33.1.1.190 will this now work? I couldnt find a definitive post in the Release Notes other than this for 33.1.1.86:
_AIR-395: Fix for remote notification event token format in iOS 13.0 (Gamua-263)_

Thanks for any help you can give.

**Comment by FliplineStudios** [comment]
That's odd, in the past few days I've submitted and released multiple updates to iOS apps using 33.1.1.63 (with a patch for the AOT compiler for geom issues) and haven't had any issues with upload/review.  When you browse/unzip the IPA, can you check to make sure it was actually packaged with 33.1.1.63?  For mine which were accepted, in the Info.plist file I see DTPlatformBuild 17B102, DTPlatformVersion 13.2, DTXcode 1120, DTXcodeBuild 11B500, and at the very bottom CTAirSdkVersion 33.1.1.63.  Especially since we're using the same older version, just curious if Animate was actually using the 33.1 SDK in your case, and if there are any discrepancies between our Info.plist values which would explain what's happening...

**Comment by FrogDigital** [comment]
@FliplineStudios - thanks so much for your helpful reply.
I did exactly what you suggested and the Info.plist states...
<key>DTPlatformBuild</key>
        <string>16B91</string>
        <key>DTSDKBuild</key>
        <string>16B91</string>

Im not smart enough to interpret the above fully, but from your post, Im guessing that Animate is _not_ using AIR 33.1.1.63 at all then?! Which would be very weird because up until this point, Animate 2019 has been pulling in the correct AIR SDK after every AIR update I've installed.

Is that whats happening here, that Animate isnt using the correct AIR SDK?

**Comment by FrogDigital** [comment]
Good grief. I have NO idea how this happened, but when I took time to look at the AIR SDK version each of the projects were linking to within Animate.... they had somehow reset back to 32.0.x.x!?!?
I can only guess that this was due to my blundering human error.
<engage Shame Mode!>

Im sorry to have taken up your time, but huge thanks to **Flipline Studios** for giving me the massive nudge in the right direction!
